src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -1319,7 +1319,7 @@ func getDefaultOrLatestDashImage(dashImage string, dryRun bool) string {
 		return dashImage
 	}
 	dashImage = defaultDashImage
-	compatibleDashVersionsURL := fmt.Sprintf("https://raw.githubusercontent.com/pachyderm/pachyderm/master/etc/compatibility/%v", version)
```

Comment:
Not that this is up for review yet, but I don't think we typically generate dash compatibility files for custom releases, so the first `version` arg should be the version without `VersionAdditional` if (and only if) `VersionAdditional` looks like a git commit hash.

Reply:
Technically we are creating the compatibility file so we should commit it. But the instructions don't call for it, so we have not been doing it for a long time.

I'll make this change.